src/nvim/main.c

```diff
@@ -1257,6 +1257,8 @@ static void check_and_set_isatty(mparm_T *paramp)
        : (paramp->err_isatty ? OS_STDERR_FILENO : -1));
 #ifndef WIN32
   pty_process_save_termios(tty_fd);
+#else
+  (void)tty_fd;
```

Comment:
could you instead move the `#ifndef WIN32` above the declaration of `int tty_fd`?